File: r/fizzbuzz.r
for (i in c(1:100)){
  output = NULL
  flag = TRUE
  if (i %% 3 == 0)  {
    output = cbind(output, "Fizz")
    flag = FALSE
  }
  if (i %% 5 == 0)  {
    output = cbind(output, "Buzz")
    flag = FALSE
  }
  if (flag) {
    output = toString(i)
  }
  cat(output, sep = "\n")
}

File: perl/fizzbuzz.pl
use v5.10;
use warnings;

for (1 .. 100) {
    print 'Fizz' unless $_ % 3;
    print 'Buzz' unless $_ % 5;
    print "$_" if $_ % 3 && $_ % 5;
    print "\t"
}
